### PR TITLE
STR-20 -- BE: Create Role assignment randomization function

### PR DESCRIPTION
**Pull request by mbendandour**
# Description
This tasks randomly assigns a user group to a new user, and return the user group. If the user is already in the database, then it will just return the user group  
# Testing
**For PHPUnit**

- Run phpunit

- All tests should pass

**For calling function by Route**
If using Postman:

- Add Username and password fields into the Body. Use 'steve@csun.edu' for Username, and no password. 

- Verify that you are calling with a GET. Hit Send. 

- Should return "control"

# Installation
'Are there any dependencies the reviewer needs to download?'

### CODE REVIEW

File: app/Models/User.php [hunk 1]
```diff
@@ -29,4 +29,17 @@ class User extends MetaUser
     protected $hidden = [
         'password', 'remember_token',
     ];
+
+    public function userRole()
+    {
+
+        return $this->hasOne('App\Models\UserRoles','user_id');
+    }
+
```
Comment (frlzjosh) [inline app/Models/User.php]: if these comments aren't needed you should get rid of them

File: tests/phpunit/ServiceTests/UserRoleServiceTest.php [hunk 2]
```diff
@@ -5,14 +5,25 @@
...
 {
     use DatabaseMigrations;
 
+    protected $utility;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->utility = new UserRoleService();
+
```
Comment (frlzjosh) [inline tests/phpunit/ServiceTests/UserRoleServiceTest.php]: remove this dead line

File: app/Http/Controllers/LoginController.php [hunk 3]
```diff
@@ -70,4 +70,12 @@ public function logout()
        auth()->logout();
 
     }
+
+    public function sortAuthenticatedUsers()
```
Comment (liadGolan) [inline app/Http/Controllers/LoginController.php]: This same code block exists in both the login and user group controller, is it needed in both?

**Review by liadGolan — DISMISSED** [review]
This is really good. My only issue is the duplicated function that appears to be in both controllers, and appears to be only tested in one.

**Comment by liadGolan** [comment]
Actually, looking over it this is good as it is

**Review by liadGolan — APPROVED** [review]
👍

**Review by luisjg — APPROVED** [review]
👍

**Pull request merged**